src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
In my "retries" custom check I do this:

```
 class RetryRateCheck : CustomCheck, IDisposable
    {
        public RetryRateCheck(BusNotifications busNotifications) : base("RetryRateCheck", "MyCategory")
        {
            subscription = busNotifications.Errors.MessageHasBeenSentToSecondLevelRetries
                .ObserveOn(Scheduler.Default)
                .Subscribe(RegisterMessageMovedToSlr);

            clearTimer = new Timer(ResetCount);
            clearTimer.Change(interval, interval);
        }

```

What would the equivalent be in the new model?